src/ImportDefinitionsBundle/ProcessManager/ImportDefinitionsReport.php

```diff
@@ -110,6 +111,10 @@ public function generateReport(ProcessInterface $process, $log)
         }
         $items[] = 'Errors count: '.$errorsCnt;
 
+        if (isset($result['artifact'])) {
+            $items[] = 'Artifact: '. $result['artifact'];
```

Comment:
This is where ProcessManager has access to the artifact it can then do things with it like making it an asset or something else.

Reply:
BTW this class should probably be renamed as `ImportDefinitionsMonitor` since it could have other applications than just generating the report.